package.json

```diff
@@ -59,11 +59,13 @@
     "angular-oauth2-oidc": "^10.0.3",
     "blob-util": "^2.0.2",
     "bootstrap": "4.5.0",
+    "echarts": "^5.0.2",
     "express": "^4.15.2",
     "file-saver": "^2.0.2",
     "hamburgers": "^1.1.3",
     "i18next": "^19.3.4",
     "i18next-xhr-backend": "^3.2.2",
+    "ngx-echarts": "^6.0.1",
```

Comment:
Since echarts are part of policy-module which is part of financialstorefrontlib these dependencies should also be listed there as peerDependencies. Also they should be imported over schematics. (add-fsa -> index.ts, like we did for file-saver and blob-util)